akka-docs/src/test/java/jdocs/stream/operators/sourceorflow/MapAsyncs.java

```diff
@@ -99,20 +103,153 @@ private void runUnordered() {
     // #mapasyncunordered
   }
 
+  private void runPartitioned() {
+    // #mapAsyncPartitioned
+    Source<EntityEvent, NotUsed> eventsForEntities = // ...
+        // #mapAsyncPartitioned
+        Source.fromIterator(() -> Stream.iterate(1, i -> i + 1).iterator())
+            .throttle(1, Duration.ofMillis(1))
```

Comment:
Not sure using throttle at 1.millis makes a lot of sense, maybe just skip that (even if not included in sample)

Reply:
the throttle is (as I remember it) primarily there to keep the log ordering sensible.